sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java

```diff
@@ -109,7 +110,7 @@ public SensitivityAnalysisParameters setFlowVoltageSensitivityValueThreshold(dou
     /**
      * FlowVoltageSensitivityValueThreshold is the threshold under which sensitivity values having
      * variable type among INJECTION_REACTIVE_POWER and BUS_TARGET_VOLTAGE
-     * and function type among BUS_VOLTAGE, BRANCH_REACTIVE_POWER_1/2/3 and BRANCH_CURRENT_1/2/3
```

Comment:
No error here too, maybe just some clarifications...

Reply:
FlowVoltageSensitivityValueThreshold is the threshold under which sensitivity values having variable type among INJECTION_REACTIVE_POWER and function type among BUS_VOLTAGE or variable type among BUS_TARGET_VOLTAGE and function type among BRANCH_REACTIVE_POWER_1/2/3 and BRANCH_CURRENT_1/2/3.
Is it better for you?

Reply:
yes!